### ISSUE TITLE
Installation from CPAN is broken

### ISSUE THREAD

**Issue by ap** [muir/Text-Tabs-Wrap#5]
This stanza in `Makefile.PL` is broken:

https://github.com/muir/Text-Tabs-Wrap/blob/b46f58dcede3b1a3e9ed43bceb2b030a4003e280/Makefile.PL#L24-L33

Because:

````
$ grep ^INST_LIBDIR Makefile
INST_LIBDIR      = $(INST_LIB)/Text
$ grep ^INST_LIB\  Makefile
INST_LIB = blib/lib
````

Meaning that `$(INST_LIBDIR)/Text/Tabs.pm` expands out to <code>blib/lib/<em>Text/Text</em>/Tabs.pm</code>, i.e. the module installs as though its name were Text::Text::Tabs.

The tests pass anyhow because they have not changed significantly in a long time, so that the version of Text::Tabs that has been shipping with perl for quite some time has always been able to pass the tests of a newer version of the module. So the module gets installed to the wrong place, then the tests load the non-upgraded in-core version of the module, and so the tests succeed and installation looks as though it worked.

In other words, upgrading Text::Tabs from CPAN hasn’t worked in 6 years, and nobody noticed.

**Comment by ap** [comment]
I’m fixing this now over in my fork.

**Comment by ap** [comment]
> In other words, upgrading Text::Tabs from CPAN hasn’t worked in 6 years, and nobody noticed.

I have to correct myself. My conclusion was wrong, because I didn’t realise the whole story. It was only the penultimate commit that broke installation of the module, but this patch never made it to CPAN.

The `PM` stanza as written is not really correct, but it worked fine so long as the `NAME` value was also wrong. `INST_LIBDIR` is defined to hold the path up to the last segment, and before the change of `NAME` to `Text::Tabs`, there were no extra segments. So its value was identical to `INST_LIB`. Thus everything ended up working out: it was subtly wrong in two places, but in such a way that one wrong cancelled the other.

The second to last commit fixed the `NAME` value, and that exposed the incorrectness in the `PM` stanza as a bug. So that commit needed to change both keys.
